.circleci/config.yml

```diff
@@ -36,9 +36,9 @@ jobs:
           aws-access-key-id: AWS_ACCESS_KEY_ID
           aws-region: AWS_DEFAULT_REGION
           aws-secret-access-key: AWS_SECRET_ACCESS_KEY
-          from: build/latest
+          from: build/
           overwrite: false
-          to: "s3://ui.wundery.com/v1.0.13" # update version number here.
+          to: "s3://ui.wundery.com"
```

Comment:
@mgorabbani  Good job! I have no blockers here; You could merge it and continue with epic work